ext/fetch/23_request.js

```diff
@@ -161,7 +161,7 @@
   function cloneInnerRequest(request) {
     const headerList = [
       ...new SafeArrayIterator(
-        ArrayPrototypeMap(request.headerList, (x) => [x[0], x[1]]),
+        ArrayPrototypeMap(request.headerList || [], (x) => [x[0], x[1]]),
```

Comment:
This seems off... In what situation would `request.headerList` be undefined?

Reply:
thats a good quesiton. Maybe i'm treating the symptom and not the root cause

Reply:
not sure if this is correct but adding ` headerList: headersCb()`to `fromFlashRequest` might be the real fix?